server.go

```diff
@@ -262,11 +270,32 @@ func (s *Server) ReadSocket(packetPool *sync.Pool, reuseport bool) {
 
 // HTTPServe starts the HTTP server and listens perpetually until it encounters an unrecoverable error.
 func (s *Server) HTTPServe() {
+	var prf interface {
+		Stop()
+	}
+
+	once := sync.Once{}
```

Comment:
rename to `profileStopOnce`? now that we've got `profileStartOnce`, the name is a bit ambiguous, and i initially thought you had added this one by mistake